exporters/committime/__init__.py

```diff
@@ -234,3 +234,81 @@ def commit_metric_from_build(app: str, build, errors: list) -> CommitMetric:
             setattr(metric, attr_name, value)
 
     return metric
+
+
+@attrs.frozen
+class GitRepo:
```

Comment:
This is a replace of [CommitMetric](https://github.com/dora-metrics/pelorus/blob/master/exporters/committime/__init__.py#L39) right?

If so, please checks the changes here https://github.com/dora-metrics/pelorus/pull/900 so they are also applied here

And if you want, you can look into this issue https://github.com/dora-metrics/pelorus/issues/903

Reply:
It is not a replacement, but is part of the [replacement input](https://github.com/dora-metrics/pelorus/pull/720/files#diff-2793729d524eeb1428c47da0196219fc76db6a641df388fa295fadf1630bf582R91).

Mind if I ask a few more questions in #900?